utils/src/treev2/quickPickWizard/ContextValueQuickPickStep.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { TreeItem } from 'vscode';
+import * as types from '../../../index';
+import { parseContextValue } from '../../utils/contextUtils';
+import { PickFilter } from './common/PickFilter';
+import { GenericQuickPickOptions, GenericQuickPickStep } from './GenericQuickPickStep';
+
+export interface ContextValueFilterQuickPickOptions extends GenericQuickPickOptions {
+    contextValueFilter: types.ContextValueFilter;
+}
+
+export class ContextValueQuickPickStep<TContext extends types.QuickPickWizardContext, TOptions extends ContextValueFilterQuickPickOptions> extends GenericQuickPickStep<TContext, TOptions> {

```

Comment:
I think the naming difference is OK since the `FindByIdQuickPickStep` is not used for "pick" experiences. It doesn't show any UI and is only used to "find" items.

Also I'm not sure the find by id stuff is going to stick around. It may end up being implemented directly on the TDP in resource groups.